irohad/ametsuchi/impl/postgres_query_executor.cpp

```diff
@@ -449,41 +450,100 @@ namespace iroha {
 
     QueryExecutorResult PostgresQueryExecutorVisitor::operator()(
         const shared_model::interface::GetAccountTransactions &q) {
-      using QueryTuple =
-          QueryType<shared_model::interface::types::HeightType, uint64_t>;
+      using QueryTuple = QueryType<shared_model::interface::types::HeightType,
+                                   uint64_t,
+                                   uint64_t>;
       using PermissionTuple = boost::tuple<int>;
 
-      auto cmd = (boost::format(R"(WITH has_perms AS (%s),
+      auto &pagination_info = q.paginationMeta();
```

Comment:
If `paginationMeta()` returns constref, it should be `const auto &pagination_info`